src/Element/SoundCloudEmded.tsx

```diff
@@ -0,0 +1,16 @@
+const SoundCloudEmbed =  ({link}:  {link: string}) => {
+
+    return(
+        <>
```

Comment:
Fragment is not needed if its a single element

Reply:
https://github.com/v0l/snort/pull/112/commits/ea61b01f375a6fbd7b29ebc4af6306555ceb07d0